### PR TITLE
feat(sat_feedback): improve feedback page

### LINKED ISSUE #49: Mejorar la página de feedback text
Actualmente la página de feedback text es esta:
imatge

@nadning he preparado con mucho amor el siguiente mockup:
mockup_feedback_text_sat

### PR DESCRIPTION
**Pull request by agustibr**
## Description

This PR adds:
- a new route for the SAT Results
- changes in the feedback page design
- adds :subtitle and :effort attributes to results
- [dev] overwrites some Form cells
- [dev] adds a `.editorconfig` to the repo (same as Decidim)

## Related

- fixes #49 

## Screenshots

Screenshot 2022-04-08 at 20-58-37 Admin - Denesik Inc

Screenshot 2022-04-08 at 20-57-58 Survey - aaQuae doloribus harum neque blanditiis  - Denesik Inc

### CODE REVIEW

File: app/cells/indices/sat_feedback/show.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<div class="column medium-6" data-relevance="<%= model.score %>" data-matching="<%= model.matching %>">
+  <div class="card">
+    <div class="row card-section">
+      <div class="column small-3">
+        <img alt="member-avatar" src="/decidim-packs/media/images/default-avatar-aaa9e55bac5d7159b847.svg">
```
Comment (microstudi) [inline app/cells/indices/sat_feedback/show.erb:5]:
```suggestion
        <%= image_pack_tag "media/images/default-avatar.svg", alt: "member-avatar" %>
```
shoudn't be this something like this?

File: config/initializers/decidim_overwrites.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Rails.application.config.to_prepare do
+  Decidim::Surveys::SurveysController.prepend Decidim::Overwrites::Surveys::SurveysController
+  Decidim::Forms::MatrixReadonlyCell.prepend Decidim::Overwrites::Forms::MatrixReadonlyCell
+  Decidim::Forms::AnswerReadonlyCell.prepend Decidim::Overwrites::Forms::AnswerReadonlyCell
```
Comment (microstudi) [inline config/initializers/decidim_overwrites.rb:6]: I think we shoud add these files to the md5 checker in spec/lib/overrides_spec.rb

**Review by microstudi — APPROVED** [review]
Nice job @agustibr !
Please pass rubocop and we merge this!

**Comment by agustibr** [comment]
Thanks @microstudi , All tests green now 🍏

**Pull request merged**